### DISCUSSION TITLE
Import/Export opml file

### DISCUSSION THREAD

**Discussion by aarcex3** [ubuntu-flutter-community/musicpod#978] [Q&A] [answered]
Hi, I just discovered this app, love it so far. The only issue I see is that I cannot import an opml file. Is this intended or is it a missing feature?

**Comment by Feichtmeier** [comment, marked as answer]
glad that you are enjoying it!

I need to research what this file is

The design idea for this app is:

- local audio: have a propertly tagged local audio collection, then use musicpod, musicpod does the rest
- radio: use the radio search to create your station lib, musicpod does the rest
- podcast: use the podcast search to create your favs, musicpod does the rest

the manual file import is somehow there, but it is not really a feature I added much love to (yet). I can see if I find time for such file type import in the future. We could also convert this into an issue if you give me a little bit more context : )

**Comment by brunmer4** [comment]
Hi,
Thanks for the app, just installed version 3.0.
As warned, my podcasts and radio stations did not carry over from the previous version, so I exported them before upgrading into opml files. Now I seem to be unable to find a way to import them. Do I miss something?
Cheers!

**Reply by Feichtmeier** [reply]
Hi in und settings:)

**Reply by Feichtmeier** [reply]
That little arrow... I know I know it needs to be better ui... Next minor release

**Reply by brunmer4** [reply]
Got it. Thanks!
